config.yaml

```diff
@@ -12,6 +12,75 @@ ignoreFiles:
 taxonomies:
   tag: ""
   category: ""
+menu:
+  main:
+    - identifier: features
+      name: Features
+      url: https://open-emr.org/wiki/index.php/OpenEMR_Features
```

Comment:
shouldn't these be relative (in case running in dev environment)
```
/wiki/index.php/OpenEMR_Features
```

Reply:
Perhaps. But I see it as 2 separate websites - marketing and docs. If we don't put the full URL in, it breaks in dev environments. Since it truly is a completely backend and site (that just happens to sit in a sub folder) I think keeping them like this is best.

No right or wrong answer on this, no Best Practice - just a design choice

Reply:
only noted this since the wiki link below is relative. shouldn't they be consistent?

Reply:
That's my oversight. Will make it an absolute path